atlasdb-dbkvs-tests/src/test/resources/docker-startup/01_create_user.sql

```diff
@@ -0,0 +1,15 @@
+CREATE USER palantir IDENTIFIED BY "7_SeeingStones_7";
+
+GRANT
+    CREATE SESSION,
+    CREATE PROCEDURE,
+    CREATE SEQUENCE,
+    CREATE TABLE,
+    CREATE VIEW,
+    CREATE TYPE
+TO palantir;
+
+ALTER USER palantir
+    QUOTA UNLIMITED ON users;
```

Comment:
it's also possible to put this clause in the initial create user so you don't need to alter it after.